src/api/dashboard.js

```diff
@@ -45,11 +45,17 @@ Dashboard.prototype = {
     var initialState = this._dashboard.dashboardView.getInitialMapState();
     var currentCenter = this._dashboard.vis.map.get('center');
     var currentZoom = this._dashboard.vis.map.getZoom();
-    if (_.isEqual(currentCenter, initialState.center) && _.isEqual(currentZoom, initialState.zoom)) return {};
-    return {
-      center: currentCenter,
-      zoom: currentZoom
-    };
+    var upperLat = initialState.center[0] * 0.01;
```

Comment:
I'd be easier for me to understand what this change is about if there was a test for it...